src/utils/session-helper.js

```diff
@@ -272,6 +274,48 @@ const createFileChecksum = async (fileObject) => {
   return sha256(text);
 };
 
+/**
+ * Helper Function that returns Driver's License ttl file based off of image passed
+ *
+ * @memberof utils
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * @param {string} documentUrl - url of uploaded document or resource
+ */
+
+const createDriversLicenseTtlFile = async (fileObject, documentUrl, checksum) => {
+  const dlData = await getDriversLicenseData(fileObject.file);
+  return buildThing(createThing({ name: 'document' }))
+    .addDatetime(RDF_PREDICATES.uploadDate, new Date())
+    .addStringNoLocale(RDF_PREDICATES.additionalType, dlData.DCA)
+    .addStringNoLocale(RDF_PREDICATES.conditionsOfAccess, dlData.DCB)
+    .addStringNoLocale(RDF_PREDICATES.expires, `${formattedDate(dlData.DBA)}`)
+    .addStringNoLocale(RDF_PREDICATES.givenName, dlData.DCS)
+    .addStringNoLocale(RDF_PREDICATES.alternateName, dlData.DAC)
+    .addStringNoLocale(RDF_PREDICATES.familyName, dlData.DAD)
+    .addStringNoLocale(RDF_PREDICATES.dateIssued, `${formattedDate(dlData.DBD)}`)
+    .addStringNoLocale(RDF_PREDICATES.dateOfBirth, `${formattedDate(dlData.DBB)}`)
+    .addStringNoLocale(RDF_PREDICATES.gender, dlData.DBC)
+    .addStringNoLocale(RDF_PREDICATES.Eye, dlData.DAY)
+    .addStringNoLocale(RDF_PREDICATES.height, dlData.DAU)
```

Comment:
For height, I believe this is in inches. The extra 0 in front is probably for the extremely rare case of people taller than 8ft 4in, but I'm not sure.

In either case, I think we can use .addInteger instead of .addStringNoLocale

Reply:
This seemed like a simple request, but I'm getting that 'Operation not ...' error message ending in '[object Object]'. I've tried coercing the dlData.DAU to a Number(), and even JSON.stringify() just to see if either of those would work with addInteger, but apparently not... I'm not sure.

Reply:
Same feedback as above with .addDate, seems strange that it's not working. I'll do a little testing myself to see what might be the problem.

Reply:
For .addInteger, I seem to get it working with just using Number(dlData.DAU).

![Screen Shot 2023-05-18 at 2 24 36](https://github.com/codeforpdx/PASS/assets/14917816/e13f9083-4857-454d-b00b-abd5c3b306fd)

Reply:
dang...I swear I tried that, and am not sure....I'll test it again, maybe that was coming from the addDate() then...

Reply:
Alright, I tested both, and have pushed these new changes. Thanks for pulling the hard work on that one. As I said in the other Date issue, if you could point me to the docs so I can review them for possible issues in the future. Thanks so much!!!